docs/_includes/errormessages.adoc

```diff
@@ -0,0 +1,264 @@
+
+[appendix]
+== Error Messages
+
+// use <source> for an asciidoc source file with line number
+// use <file> for an arbitrary file
+// use <x> for anything else
+
+////
+Testing was carried out with
+Asciidoctor 1.5.5 [http://asciidoctor.org]
+Runtime Environment (ruby 2.3.1p112 (2016-04-26) [i386-linux-gnu]) (lc:UTF-8 fs:UTF-8 in:- ex:UTF-8)
+////
+
+The most common error and warning messages are listed alphabetically in the table below.
+
+[NOTE]
+====
+These messages are produced by asciidoctor and asciidoctor-ant.
+If you are using asciidoctorj or the asciidoctor API, you may see different errors.
+====
+
+.Common Error Messages
+[options="header,breakable",cols="<20,<60,<50,<30"]
+|====
+|Category (1)|Message |Cause |See also
+
+|ERROR
+|input file <file> missing or cannot be read
+|Misspelled filename?
```

Comment:
Let's write this as "check that the file exists and that the filename is not misspelled"